plugins/qlcplus_4.12.1/exportTests/fixture-tool-validation.js

```diff
@@ -9,22 +9,33 @@ const mkdtemp = promisify(fs.mkdtemp);
 const writeFile = promisify(fs.writeFile);
 const execFile = promisify(require(`child_process`).execFile);
 
+const qlcplusGoboAliases = require(`../../../resources/gobos/aliases/qlcplus.json`);
+
 
 const FIXTURE_TOOL_URL = `https://raw.githubusercontent.com/mcallegari/qlcplus/master/resources/fixtures/scripts/fixtures-tool.py`;
 const FIXTURE_TOOL_DIR_PREFIX = path.join(os.tmpdir(), `ofl-qlcplus5-fixture-tool-`);
 const FIXTURE_TOOL_PATH = `resources/fixtures/scripts/fixtures-tool.py`;
 const EXPORTED_FIXTURE_PATH = `resources/fixtures/manufacturer/fixture.qxf`;
 
 /**
- * @param {Object} exportFile The file returned by the plugins' export module.
- * @param {String} exportFile.name File name, may include slashes to provide a folder structure.
- * @param {String} exportFile.content File content.
- * @param {String} exportFile.mimetype File mime type.
- * @param {Array.<Fixture>|null} exportFile.fixtures Fixture objects that are described in given file; may be omitted if the file doesn't belong to any fixture (e.g. manufacturer information).
- * @param {String|null} exportFile.mode Mode's shortName if given file only describes a single mode.
+ * @typedef {Object} ExportFile
+ * @property {String} name File name, may include slashes to provide a folder structure.
+ * @property {String} content File content.
+ * @property {String} mimetype File mime type.
+ * @property {Array.<Fixture>|null} fixtures Fixture objects that are described in given file; may be omitted if the file doesn't belong to any fixture (e.g. manufacturer information).
+ * @property {String|null} mode Mode's shortName if given file only describes a single mode.
+ */
+
+/**
+ * @param {ExportFile} exportFile The file returned by the plugins' export module.
+ * @param {Array.<ExportFile>} allExportFiles An array of all export files.
  * @returns {Promise.<undefined, Array.<String>|String>} Resolve when the test passes or reject with an array of errors or one error if the test fails.
  */
-module.exports = async function testFixtureToolValidation(exportFile) {
+module.exports = async function testFixtureToolValidation(exportFile, allExportFiles) {
```

Comment:
Please change this also in the other export tests.

Reply:
I didn't want to do this in this PR to prevent all export tests from running; but they run anyway since the schema changed :sweat_smile: